packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -132,14 +136,22 @@ interface PendingBlob {
 	blob: ArrayBufferLike;
 	status: PendingBlobStatus;
 	storageId?: string;
-	handleP: Deferred<IFluidHandle<ArrayBufferLike>>;
+	handleP: Deferred<BlobHandle>;
 	uploadP?: Promise<ICreateBlobResponse>;
 	uploadTime?: number;
 	minTTLInSeconds?: number;
+	attached: boolean;
+	acked: boolean;
 }
 
 export interface IPendingBlobs {
-	[id: string]: { blob: string; uploadTime?: number; minTTLInSeconds?: number };
+	[id: string]: {
+		blob: string;
+		uploadTime?: number;
+		minTTLInSeconds?: number;
+		attached: boolean;
```

Comment:
do we actually need to persist these? if so, i think they need to be optional, as previously persisted state won't have them